web-ui/src/main/resources/catalog/components/viewer/wfsfilter/WfsFilterService.js

```diff
@@ -195,12 +276,12 @@
        * @param {string} wfsUrl url of the wfs service
        */
       this.getApplicationProfile = function(uuid, ftName, wfsUrl, protocol) {
-        return $http.post('../api/records/' + uuid +
-            '/query/wfs-indexing-config', {
-              url: wfsUrl,
-              name: ftName,
-              protocol: protocol
-            });
+        return $http.post('../api/0.1/records/' + uuid +
```

Comment:
Do not reintroduce version number in API path. In version 4, the API version is GN version. Version in path has been removed.

Reply:
noted, thanks